zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java

```diff
@@ -84,18 +85,18 @@ public ReportLastScan() {}
 
     /**
      * @deprecated generate has been deprecated in favor of using {@link #generate(String fileName,
-     *     Model model, ReportType reportType)}
+     *     ReportType reportType)}
      */
     @Deprecated
-    public File generate(String fileName, Model model, String xslFile) throws Exception {
+    public File generate(String fileName, String xslFile) throws Exception {
         StringBuilder sb = new StringBuilder(500);
-        this.generate(sb, model);
+        this.generate(sb);
         return ReportGenerator.stringToHtml(sb.toString(), xslFile, fileName);
     }
 
-    public File generate(String fileName, Model model, ReportType reportType) throws Exception {
```

Comment:
We need to keep binary compatibility, for the add-ons.

Reply:
Force pushed required changes.

Reply:
Sorry I was not clear, the comment applies to the other public methods as well. We should deprecate the public methods with unused parameter and add new ones without the parameter (the initial commit was just missing the old/deprecated methods).

Reply:
My bad. Should be fine now.